docs/sources/configuration/_index.md

```diff
@@ -2222,7 +2222,7 @@ The `limits_config` block configures global and per-tenant limits in Loki.
 # a query request. If a specific per-tenant timeout is used, this timeout is
 # ignored.
 # CLI flag: -querier.query-timeout
-[query_timeout: <duration> | default = 1m]
```

Comment:
the default `querier.query-timeout` should be 1m, could you link where you find it being `0s`?

Reply:
In the docs about upgrading Loki to 2.7.0.

https://grafana.com/docs/loki/latest/upgrading/#270 

If you scroll a little there is written: 

"Querier query_timeout default changed
The previous default value for [querier.query_timeout] of [1m] has changed to [0s]."

Reply:
Good catch, thank you. That docs page for `latest` is wrong/outdated, if you see it for (next)[https://grafana.com/docs/loki/next/upgrading/] (which is what is currently on main) you'll see that the entry was removed.

That said, it is our mistake: we forgot to backport the change to the 2.7 docs.